proxy/api/src/identity.rs

```diff
@@ -39,9 +39,38 @@ pub struct Identity {
 
 impl From<(PeerId, Person)> for Identity {
     fn from((peer_id, user): (PeerId, Person)) -> Self {
-        let urn = user.urn();
-        let handle = user.subject().name.to_string();
-        let ethereum = match user.payload().get_ext::<EthereumClaimExtV1>() {
+        let identity = RemoteIdentity::from(user);
+        let shareable_entity_identifier = Identifier {
+            handle: identity.metadata.handle.clone(),
+            peer_id,
+        };
+        Self {
+            peer_id,
+            urn: identity.urn,
+            shareable_entity_identifier,
+            metadata: identity.metadata,
+            avatar_fallback: identity.avatar_fallback,
+        }
+    }
+}
+
+/// The remote users personal identifying metadata and keys.
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RemoteIdentity {
```

Comment:
Can we rename this to `Person`? This is the terminology from radicle link. `Identity` is an Upstream-only concept because it includes `peer_id` which is not present in the link data.

Reply:
Fixed